app/src/sharedTest/java/org/oppia/app/player/audio/AudioFragmentTest.kt

```diff
@@ -295,6 +284,10 @@ class AudioFragmentTest {
     @GlobalLogLevel
     @Provides
     fun provideGlobalLogLevel(): LogLevel = LogLevel.VERBOSE
+
+    @CacheAssetsLocally
```

Comment:
<img width="921" alt="Screenshot 2019-11-28 at 11 59 17 PM" src="https://user-images.githubusercontent.com/9396084/69826902-2157e400-123b-11ea-9cf7-e49c48857f49.png">

Was not able run these test because of this error.

Reply:
Merge error?